docs/trino-connector/trino-cascading-query.md

```diff
@@ -0,0 +1,162 @@
+---
+title: "Apache Gravitino connector Trino cascading query"
+slug: /trino-connector/trino-cascading-query
+keyword: gravitino connector trino
+license: "This software is licensed under the Apache License version 2."
+---
+
+# Background
+With the `Apache Gravitino Trino connector` and the `Gravitino Trino cascading connector`, you can implement cascading queries in Trino.
```

Comment:
Where can one find the `Apache Gravitino Trino connector`? How can a user access it? If it is inaccessible, I recommend removing this documentation as it holds no significance for the user.